From 1be16306f62c66a2d58df0f5cee1a4d0ae995ec5 Mon Sep 17 00:00:00 2001
From: Wen He <wen.he_1@nxp.com>
Date: Thu, 31 Jan 2019 15:55:44 +0800
Subject: [PATCH 625/706] arm64: dts: ls1028aqds: Add resolution property

This patch add resolution property for LS1028AQDS board. It will
change the supported resolution list of the specify monitor.

Signed-off-by: Wen He <wen.he_1@nxp.com>
(cherry picked from commit 4f889c110bba4310f496bb057058ada79961b5d8)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts b/arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts
index e26cdb36f708..49fd654ecad0 100755
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a-qds.dts
@@ -451,6 +451,10 @@
 
 &edp {
 	fsl,no_edid;
+	resolution = "3840x2160@60",
+		   "1920x1080@60",
+		   "1280x720@60",
+		   "720x480@60";
 	lane_mapping = <0x4e>;
 	edp_link_rate = <0x6>;
 	edp_num_lanes = <0x4>;
-- 
2.17.1

